Containerfile-backend

```diff
@@ -23,30 +23,30 @@ RUN     apk update && \
 
 # Install system dependencies
 RUN     apk add --no-cache \
-            php81 \
-            php81-ctype \
-            php81-fpm \
-            php81-exif \
-            php81-fileinfo \
-            php81-gd \
-            php81-iconv \
-            php81-ldap \
-            php81-pdo_sqlite \
-            php81-simplexml \
-            php81-tokenizer \
-            php81-phar \
-            php81-curl \
-            php81-mbstring \
-            php81-openssl \
-            php81-zip \
-            php81-intl
+            php8 \
+            php8-ctype \
+            php8-fpm \
+            php8-exif \
+            php8-fileinfo \
+            php8-gd \
+            php8-iconv \
+            php8-ldap \
+            php8-pdo_sqlite \
+            php8-simplexml \
+            php8-tokenizer \
+            php8-phar \
+            php8-curl \
+            php8-mbstring \
+            php8-openssl \
+            php8-zip \
+            php8-intl
 
 # Configure directory permissions
-RUN     chown www-data /var/log/php81 && \
+RUN     chown www-data /var/log/php8 && \
```

Comment:
To confirm: can this log directory configuation (and the associated volume definitions) be removed, now that #195 has been applied?

Reply:
Yep, it appears so.  That can be handled in a follow-up pull request.